Index: node_modules/eslint-plugin-tailwindcss/lib/rules/no-contradicting-classname.js
===================================================================
--- node_modules/eslint-plugin-tailwindcss/lib/rules/no-contradicting-classname.js
+++ node_modules/eslint-plugin-tailwindcss/lib/rules/no-contradicting-classname.js
@@ -200,8 +200,9 @@
 
     const scriptVisitor = {
       JSXAttribute: attributeVisitor,
       TextAttribute: attributeVisitor,
+      SvelteAttribute: attributeVisitor,
       CallExpression: callExpressionVisitor,
       TaggedTemplateExpression: function (node) {
         if (!tags.includes(node.tag.name ?? node.tag.object?.name ?? node.tag.callee?.name)) {
           return;
