Index: node_modules/eslint-plugin-tailwindcss/lib/rules/migration-from-tailwind-2.js
===================================================================
--- node_modules/eslint-plugin-tailwindcss/lib/rules/migration-from-tailwind-2.js
+++ node_modules/eslint-plugin-tailwindcss/lib/rules/migration-from-tailwind-2.js
@@ -92,9 +92,9 @@
       let suffix = '';
       if (arg === null) {
         originalClassNamesValue = astUtil.extractValueFromNode(node);
         const range = astUtil.extractRangeFromNode(node);
-        if (node.type === 'TextAttribute') {
+        if (node.type === 'TextAttribute' || node.type === 'SvelteAttribute') {
           start = range[0];
           end = range[1];
         } else {
           start = range[0] + 1;
@@ -287,8 +287,9 @@
 
     const scriptVisitor = {
       JSXAttribute: attributeVisitor,
       TextAttribute: attributeVisitor,
+      SvelteAttribute: attributeVisitor,
       CallExpression: callExpressionVisitor,
       TaggedTemplateExpression: function (node) {
         if (!tags.includes(node.tag.name ?? node.tag.object?.name ?? node.tag.callee?.name)) {
           return;
